packages/devtools/lib/src/ui/ui_utils.dart

```diff
@@ -75,19 +76,30 @@ List<CoreElement> getServiceExtensionButtons() {
   ];
 }
 
-CoreElement createTipsButton(String pageName) {
-  return PButton.icon(
-    '$pageName Docs',
-    const MaterialIcon(
-      'description',
-      Colors.black45,
-    ),
-    title: 'Documentation on using the $pageName page',
-    classes: ['btn-outline'],
-  )
-    ..small()
-    ..click(() => html.window.open(
-        'https://flutter.github.io/devtools/${pageName.toLowerCase()}', null));
+StatusItem createDocumentationStatusItem(String pageName) {
+  final icon = createIconElement(const MaterialIcon(
+    'description',
+    Colors.black45,
+  ));
+  // TODO(jacobr): add this style to the css for all icons displayed as HTML
+  // once we verify there are not unintended consequences.
+  icon.element.style
+    ..verticalAlign = 'text-bottom'
+    ..marginBottom = '0';
+  final element = CoreElement('a')
```

Comment:
When we hover over the links, they do change their appearance appropriately?

Reply:
the links turn blue on hover.

Reply:
it looks like links are intentionally greyed when not hovering which looks slick but . is something we should evaluate the usability of as part of when do some simple studies of how this tool is used.